arches/app/media/js/utils/resource.js

```diff
@@ -1,4 +1,5 @@
 import arches from 'arches';
+import resourceLookup from 'viewmodels/map'
```

Comment:
instead of import from viewmodels/map, the function should be refactored to take `resourceLookup` as an arg. `resource-instance-select.js` will then need to be updated to have it declared before the call